desktop/apps/auction2/components/Layout.js

```diff
@@ -0,0 +1,59 @@
+import AuctionBlock from 'desktop/components/react/auction_block/auction_block'
+import Banner from 'desktop/apps/auction2/components/page/Banner'
+import CommercialFilter from './commercial_filter'
+import Footer from 'desktop/apps/auction2/components/page/Footer'
+import Header from 'desktop/apps/auction2/components/page/Header'
+import PropTypes from 'prop-types'
+import React from 'react'
+import WorksByFollowedArtists from 'desktop/apps/auction2/components/commercial_filter/WorksByFollowedArtists'
+import { connect } from 'react-redux'
+
+function Layout (props) {
+  const { articles, auction, displayFollowedArtistsRail } = props
+  const { associated_sale, eligible_sale_artworks_count } = auction.toJSON()
```

Comment:
So the idea is to get to a raw data representation early, so that the rest of the logic isn't coupled directly to Backbone?